cli/jobs/single-step/r/accidents/src/aml_utils.R

```diff
@@ -0,0 +1,92 @@
+# AML utility to extend support for MLflow in Azure Machine Learning (AML). This utility does the following:
+# 1. Understands AML MLflow tracking url by extending OSS MLflow R client.
```

Comment:
Please use "Azure ML" and not "AML" for the shortened form

Reply:
Done